src/blueprints/tv2/factories/tv2-video-mixer-action-factory.ts

```diff
@@ -11,11 +9,11 @@ import { PieceType } from '../../../model/enums/piece-type'
 import { Tv2SourceLayer } from '../value-objects/tv2-layers'
 import { TransitionType } from '../../../model/enums/transition-type'
 
-export class Tv2VideoSwitcherActionFactory {
-  constructor(private readonly videoSwitcherTimelineObjectFactory: Tv2VideoSwitcherTimelineObjectFactory) {
+export class Tv2VideoMixerActionFactory {
```

Comment:
Something feels wrong about having a `VideoMixerActionFactory`. It's too broad and it doesn't reflect any particular type of Piece or Action.
Case and point: `VideoMixerActionFactory` indicates that all Actions that needs to send data to a VideoMixer should be created in here, but all our TransitionActions are being created by our `TransitionActionFactory`. So we already have "VideoMixerActions" created elsewhere than our `VideoMixerActionFactory`.

We should either rename our `VideoMixerActionFactory` something else or move the transition from `TransitionActionFactory` in here.

Reply:
I don't think that VideoMixerActionFactory is way off. What we want to create actions for is essentially for configuring the video mixer. So it could be renamed to `Tv2VideoMixerConfigurationActionFactory`.

Reply:
Has been renamed to `Tv2VideoMixerConfigurationActionFactory`.